build/generate-manifests.sh

```diff
@@ -35,7 +35,7 @@ else
   DOCUMENTATION_URL="https://docs.openshift.com/container-platform/latest/service_mesh/v2x/servicemesh-release-notes.html"
   BUG_URL="https://issues.redhat.com/projects/OSSM"
   OLM_FEATURES="[\"Disconnected\",\"fips\"]"
-  ARCHITECTURE_LABELS=$(generateArchitectureLabels amd64 s390x ppc64le)
+  ARCHITECTURE_LABELS=$(generateArchitectureLabels amd64 s390x ppc64le arm64)
```

Comment:
Should it be `aarch64` instead?

Reply:
No, arm64 is correct here.  General rule is whenever `amd64` is used instead of `x86_64`, `arm64` should be used instead of `aarch64`.

Reply:
yeah, definitely not aarch64 in the CSV. That value comes from the GOARCH env var... `arm64`.

Everything in our build infrastructure (CpaaS etc) should be `aarch64` though